keps/sig-windows/2258-node-service-log-viewer/README.md

```diff
@@ -247,43 +257,77 @@ to see logs across nodes from the same time. Similarly for pods, it will allow
 seeing logs across containers aligned by time.
 
 Given that the feature will be introduced behind a feature gate, by default
-`kubectl logs nodes` will return a feature not enabled message. When the
-feature is enabled in alpha phase, `kubectl logs nodes` will display a
-warning message that the feature is in alpha. When the `--service` option
+`kubectl node-logs` will return a feature not enabled message. When the
+feature is enabled in alpha phase, `kubectl node-logs` will display a
```

Comment:
How are you planning on differentiating between feature disabled vs a cluster where this feature does not exists (ie. older version). I doubt you have that option, so it's safest to say that this functionality is not available.

Reply:
Addressed